### PR TITLE
[feat] add notification toast

### PR DESCRIPTION
**Pull request by junaidshaikh-js**
This PR includes

- Toast notifications
- Context for toast messages 
- The preview will be available in the next PR when the toast notification will be added to the authentication.

### CODE REVIEW

File: src/context/toast-context/toast-context.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+};
+
+const ToastContext = createContext(initialValue);
+
+const ToastProvider = ({ children }) => {
+  const [toastMessage, setToastMessage] = useState(initialValue);
+  const [timerId, setTimerId] = useState(null);
+
+  useEffect(() => {
+    clearTimeout(timerId);
```
Comment (Maianki) [inline src/context/toast-context/toast-context.js:15]: query: Why are you not clearing timeout in cleanup?

**Review by Maianki — APPROVED** [review]
Everything looks good. Your code is very readable and well organized.

**Review by amarphule — APPROVED** [review]
code is well-formatted and well structured. nice work

File: src/context/toast-context/toast-context.js (hunk 1 above)
Comment (junaidshaikh-js) [reply src/context/toast-context/toast-context.js:15]: For no particular reason.

**Pull request merged**